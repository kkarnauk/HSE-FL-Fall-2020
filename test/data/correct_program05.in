f :- fffff fff, ffff, ffff, hello, my friend, how (are you) 
(what are you doing), h; shdshdg. f. f.
