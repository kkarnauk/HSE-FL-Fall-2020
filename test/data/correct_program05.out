|relation
|	|head
|	|	|identifier = 'f'
|	|body
|	|	|disjunction
|	|	|	|conjunction
|	|	|	|	|atom
|	|	|	|	|	|identifier = 'fffff'
|	|	|	|	|	|identifier = 'fff'
|	|	|	|	|conjunction
|	|	|	|	|	|identifier = 'ffff'
|	|	|	|	|	|conjunction
|	|	|	|	|	|	|identifier = 'ffff'
|	|	|	|	|	|	|conjunction
|	|	|	|	|	|	|	|identifier = 'hello'
|	|	|	|	|	|	|	|conjunction
|	|	|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|	|	|identifier = 'my'
|	|	|	|	|	|	|	|	|	|identifier = 'friend'
|	|	|	|	|	|	|	|	|conjunction
|	|	|	|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|	|	|	|identifier = 'how'
|	|	|	|	|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|	|	|	|	|identifier = 'are'
|	|	|	|	|	|	|	|	|	|	|	|identifier = 'you'
|	|	|	|	|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|	|	|	|	|identifier = 'what'
|	|	|	|	|	|	|	|	|	|	|	|identifier = 'are'
|	|	|	|	|	|	|	|	|	|	|	|identifier = 'you'
|	|	|	|	|	|	|	|	|	|	|	|identifier = 'doing'
|	|	|	|	|	|	|	|	|	|identifier = 'h'
|	|	|	|identifier = 'shdshdg'

|relation
|head
|	|identifier = 'f'

|relation
|head
|	|identifier = 'f'